.circleci/config.yml

```diff
@@ -13,7 +13,6 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker:
-          version: 20.10.14
```

Comment:
had to remove this for circle ci 
<img width="1375" alt="image" src="https://github.com/user-attachments/assets/0be45670-4229-4c65-912c-991f2ab1cb6a">